frontend/src/models/auth.ts

```diff
@@ -1,22 +1,22 @@
-import Controller from '../services/Controller'
+import { clearUserCredentials, updateUserCredentials, r3 } from '../services/remote.it'
 import { IUser } from 'remote.it'
 import { createModel } from '@rematch/core'
-import { clearUserCredentials, updateUserCredentials, r3 } from '../services/remote.it'
 import { emit } from '../services/Controller'
+import Controller from '../services/Controller'
 import analyticsHelper from '../helpers/analyticsHelper'
 
 const USER_KEY = 'user'
 
 export interface AuthState {
-  loadingInitialState: boolean
+  initialized: boolean
```

Comment:
initialized state wasn't being used. Fixed that. Better startup experience. Deep links will work at initial launch through login as well as when app is open.